### ISSUE TITLE
cant set default player if only one player is installed

### ISSUE THREAD

**Issue by somedevreally** [DutchSkiddie/script.preloader#3]
Hello,

great script.

few issues i had when having installed > preset > heavy.
when only one player is selected to install it wont ask for set player as default. so next time if you play anything it will ask to choose the player. but, when multiple players are selected to install, then next dialog to set default player is avaliable. so without reason i ended up installing fen/fenlite/seren

trakt addon already have default user set, cant revoke or remove it. only way to set new user is to remove and reinstall trakt. 

and where is favorite in AH2? can it be added as tab maybe?

thanks

**Comment by DutchSkiddie** [comment]
Hi, thanks for raising the issue!

The current version is extremely buggy so I'm rewriting the entire thing as we speak.
The updated version will have a lot more room for customization and will not be suffering from this poor design.

I kind of rushed releasing this, my bad.

**Comment by somedevreally** [comment]
great, cant wait to try newer version.
any plans to for auto update? repo?

**Comment by DutchSkiddie** [comment]
> great, cant wait to try newer version.
> 
> any plans to for auto update? repo?

Yeah, as a matter of fact, when I'm done with this version, first thing on the schedule is a repo, then a custom GUI, then all extra features.
I haven't read the skinning manual yet, so had to make do with Kodi's dialog windows for now.

Extra features will include cloud integration so you don't have to transfer files.
